docs/IRISMapCubeSequence.rst

```diff
@@ -74,58 +83,58 @@ inside the object. We can see this list by doing: ::
     [
          IRISMapCube
          ---------
-         Observatory:		 IRIS
+         Observatory:		     IRIS
          Instrument:			 SJI
-         Bandpass:			 1330.0
+         Bandpass:			     1330.0
          Obs. Start:			 2018-04-26T23:07:22.780000
-         Obs. End:			 2018-04-27T01:39:47.122000
+         Obs. End:			     2018-04-27T01:39:47.122000
          Instance Start:		 2018-04-26T23:07:22.880000
-         Instance End:		 2018-04-27T01:36:40.490000
+         Instance End:		     2018-04-27T01:36:40.490000
          Total Frames in Obs.:	 49
-         IRIS Obs. id:		 3690015104
+         IRIS Obs. id:		     3690015104
          IRIS Obs. Description:	 Very large sit-and-stare 0.3x175 1s  C II   Si IV   Mg II h/k Deep x
          Cube dimensions:		 [  49. 1095. 1018.] pix
          Axis Types:			 ('time', 'custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon')
          ,
          IRISMapCube
          ---------
-         Observatory:		 IRIS
+         Observatory:		     IRIS
          Instrument:			 SJI
-         Bandpass:			 1400.0
+         Bandpass:			     1400.0
          Obs. Start:			 2018-04-26T23:07:22.780000
-         Obs. End:			 2018-04-27T01:39:47.122000
+         Obs. End:			     2018-04-27T01:39:47.122000
          Instance Start:		 2018-04-26T23:08:25.360000
-         Instance End:		 2018-04-27T01:37:42.990000
+         Instance End:		     2018-04-27T01:37:42.990000
          Total Frames in Obs.:	 49
-         IRIS Obs. id:		 3690015104
+         IRIS Obs. id:		     3690015104
          IRIS Obs. Description:	 Very large sit-and-stare 0.3x175 1s  C II   Si IV   Mg II h/k Deep x
          Cube dimensions:		 [  49. 1095. 1018.] pix
          Axis Types:			 ('time', 'custom:pos.helioprojective.lat', 'custom:pos.helioprojective.lon')
          ]
 
-
 The return value is a list with the representation of each cube inside. For an upcoming work, we
 will need to select only on cube. We can do that by indexing our sequence from 0 to N-1,
 where N is the number of cubes. To select the first cube do: ::
```

Comment:
Yes, I've ask you before but maybe the message went to an outdated review. I was asking you if we should mix the cubes and if yes in which order ? If we use time order, the wavelength will be mixed, opposite if we use wavelength (by wavelength I mean 2 different cube like cubes I used).

(rewrite out of outdated)